conan/tools/cmake/cmakedeps/templates/config.py

```diff
@@ -28,6 +28,23 @@ def additional_variables_prefixes(self):
             self.cmakedeps.get_property("cmake_additional_variables_prefixes", self.conanfile, check_type=list) or [])
         return list(set([self.file_name] + prefix_list))
 
+    @property
+    def parsed_extra_variables(self):
+        # Reading configuration from "cmake_extra_variables" property
+        from conan.tools.cmake.utils import parse_extra_variable
+        conf_extra_variables = self.conanfile.conf.get("tools.cmake.cmaketoolchain:extra_variables",
+                                                       default={}, check_type=dict)
+        dep_extra_variables = self.cmakedeps.get_property("cmake_extra_variables", self.conanfile,
+                                                          check_type=dict) or {}
+        # The configuration variables have precedence over the dependency ones
+        extra_variables = {**dep_extra_variables, **conf_extra_variables}
```

Comment:
But the ``conf_extra_variables`` shouldn't appear here, those are ``cmaketoolchain:extra_variables`` aren't they?
It is a bit unexpected that ``CMakeDeps`` define them?
I was thinking of CMakeDeps not defining the values to let the toolchain still have precedence, but I am not sure anymore